.github/workflows/main.yml

```diff
@@ -0,0 +1,61 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI Build
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
it's better to not specify master here, so that CI works in any branch; so I would replace this with: `on: [push, pull_request]`